Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -9,6 +9,8 @@
 
 namespace Mirror
 {
+    public class OnClientConnectEvent : UnityEvent<NetworkConnection> {}
```

Comment:
Add a [Serializable] tag, otherwise it will not appear in the inspector

Reply:
This is currently removed per the changes requested in Discord.